InterviewFlashCards/QueryManager.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  QueryManager.swift
+//  InterviewFlashCards
+//
+//  Created by Charles Kang on 11/2/16.
+//  Copyright © 2016 Varindra Hart. All rights reserved.
+//
+
+import Foundation
+import Firebase
+
+enum RequestType: Int {
+    case RequestTypeiOS = 1
```

Comment:
This is the last issue I need to fix... Would it be not ideal to keep this enum as an Int? What issues/disadvantages are there to use Int vs String?